functions/Invoke-DbaDiagnosticQuery.ps1

```diff
@@ -300,14 +418,33 @@ Then it will export the results to Export-DbaDiagnosticQuery.
                                 Name             = $scriptpart.QueryName
                                 Description      = $scriptpart.Description
                                 DatabaseSpecific = $scriptpart.DBSpecific
-                                DatabaseName     = $currentdb.name
+                                DatabaseName     = $currentDb
```

Comment:
This should be reverted back to `$currentDb.Name` and property updated to just `Database`